# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/test/jsmodules/chrome.toml
# Commit: cc6bc73fbbfd
# Full Hash: cc6bc73fbbfd2dd796ae51f2b85c7e9c24ae2686
# Author: Alexandru Marc <amarc@mozilla.com>
# Date: 2025-09-25 05:12 +0300
# Regressor Bug: 1968890
# File Overlap Count: 14
# Overlapping Files: dom/base/test/jsmodules/chrome.toml, dom/base/test/jsmodules/test_dynamicImportErrorMessage.html, dom/script/ScriptLoadContext.cpp, dom/script/ModuleLoader.cpp, dom/base/test/jsmodules/import_404.mjs
# Description:
#   Revert "Bug 1968890 - Part 9: Add test for dynamic import. r=jonco" for causing multiple wpt failures
# ==============================================================================

diff -r e1c5be04b3a0 -r cc6bc73fbbfd dom/base/test/jsmodules/chrome.toml
--- a/dom/base/test/jsmodules/chrome.toml	Wed Sep 24 23:17:54 2025 +0000
+++ b/dom/base/test/jsmodules/chrome.toml	Thu Sep 25 05:12:32 2025 +0300
@@ -1,7 +1,6 @@
 [DEFAULT]
 support-files = [
   "ambiguous_export.mjs",
-  "import_404.mjs",
   "import_ambiguous.mjs",
   "import_ambiguous_indirect_export.mjs",
   "import_ambiguous_export.mjs",